test/include/test_util.hpp

```diff
@@ -105,11 +105,11 @@ check_err(const std::vector<T>& out,
     return res;
 }
 
-bool check_err(const std::vector<_Float16>& out,
-               const std::vector<_Float16>& ref,
+bool check_err(const std::vector<ck::half_t>& out,
+               const std::vector<ck::half_t>& ref,
                const std::string& msg,
-               _Float16 rtol = static_cast<_Float16>(1e-3f),
-               _Float16 atol = static_cast<_Float16>(1e-3f))
+               ck::half_t rtol = static_cast<ck::half_t>(1e-3f),
+               ck::half_t atol = static_cast<ck::half_t>(1e-3f))

```

Comment:
Doesn't this change clash with: https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/3ba149328f2704e096b2eed7ffeacff0b54fdc8b/test/include/test_util.hpp#L63 ?? If `ck::half_t` needs separate treatment then it should have it's own overload.